sdk/servicebus/azure-servicebus/azure/servicebus/management/_generated/models/_models.py

```diff
@@ -0,0 +1,440 @@
+    """
+
+    _attribute_map = {
+        'type': {'key': 'type', 'type': 'str', 'xml': {'attr': True}},
+        'topic_description': {'key': 'TopicDescription', 'type': 'TopicDescription'},
+    }
+    _xml_map = {
+        'ns': 'http://www.w3.org/2005/Atom'
+    }
+
+    def __init__(
+        self,
+        **kwargs
+    ):
+        super(CreateTopicBodyContent, self).__init__(**kwargs)
+        self.type = kwargs.get('type', "application/xml")
+        self.topic_description = kwargs.get('topic_description', None)
+
+
+class MessageCountDetails(msrest.serialization.Model):
+    """Details about the message counts in queue.
+
+    :param active_message_count: Number of active messages in the queue, topic, or subscription.
+    :type active_message_count: int
+    :param dead_letter_message_count: Number of messages that are dead lettered.
+    :type dead_letter_message_count: int
+    :param scheduled_message_count: Number of scheduled messages.
+    :type scheduled_message_count: int
+    :param transfer_dead_letter_message_count: Number of messages transferred into dead letters.
+    :type transfer_dead_letter_message_count: int
+    :param transfer_message_count: Number of messages transferred to another queue, topic, or
+     subscription.
+    :type transfer_message_count: int
+    """
+
+    _attribute_map = {
+        'active_message_count': {'key': 'ActiveMessageCount', 'type': 'int', 'xml': {'prefix': 'd2p1', 'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'}},
+        'dead_letter_message_count': {'key': 'DeadLetterMessageCount', 'type': 'int', 'xml': {'prefix': 'd2p1', 'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'}},
+        'scheduled_message_count': {'key': 'ScheduledMessageCount', 'type': 'int', 'xml': {'prefix': 'd2p1', 'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'}},
+        'transfer_dead_letter_message_count': {'key': 'TransferDeadLetterMessageCount', 'type': 'int', 'xml': {'prefix': 'd2p1', 'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'}},
+        'transfer_message_count': {'key': 'TransferMessageCount', 'type': 'int', 'xml': {'prefix': 'd2p1', 'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'}},
+    }
+    _xml_map = {
+        'ns': 'http://schemas.microsoft.com/netservices/2011/06/servicebus'
+    }
+
+    def __init__(
+        self,
+        **kwargs
+    ):
+        super(MessageCountDetails, self).__init__(**kwargs)
+        self.active_message_count = kwargs.get('active_message_count', None)
+        self.dead_letter_message_count = kwargs.get('dead_letter_message_count', None)
+        self.scheduled_message_count = kwargs.get('scheduled_message_count', None)
+        self.transfer_dead_letter_message_count = kwargs.get('transfer_dead_letter_message_count', None)
+        self.transfer_message_count = kwargs.get('transfer_message_count', None)
+
+
+class QueueDescription(msrest.serialization.Model):
```

Comment:
We will need to create some kind of public wrapper class for this.
We could use it to separate read-only properties....

Reply:
Create QueueDescription and QueueRuntimeInfo mainly because we combined the previous generated QueueDescription and QueueRuntimeInfo in swagger. Now the convenience layer separate them from one internal model class.
But for read-only validation/property, I'd like to hold off for more discussions. The service also has some validation. Basic tier and standard tier may be able to change different properties.